packages/vector_graphics_compiler/pubspec.yaml

```diff
@@ -36,7 +36,7 @@ dev_dependencies:
     sdk: flutter
   test: ^1.20.1
   # See https://github.com/flutter/flutter/issues/157626
-  vector_graphics: ">=1.1.11+1 <= 1.1.12"
+  vector_graphics: ">=1.1.11+1 <= 1.1.14"
```

Comment:
1.1.14 doesn't exist yet, so we don't know if it'll be compatible under the versioning system `vector_graphics` has used previously.

Reply:
Not on pub.dev, but I just bumped it here https://github.com/flutter/packages/pull/8014/files#diff-ae3c1055bc547ec174edea702c972431d2d3d46fb9bd78a31378cacf0fd051eaR8

If I don't bump the constraints here, how is it supposed to work?

Reply:
You're right. Didn't notice someone already made vector_graphics wasm compatible. I just fixed the dev_dependency constraint to use the minimum the WASM version. Do you prefer another PR for a single line change (just dev_dependency)?

Reply:
As long as the changes aren't interdependent, they can be in a single PR. If there are still any cases where you need to both change a package, and change the *minimum* dependency on that package in another package, that must be multiple PRs because it won't compile in CI.

If you're only changing a dev dependency in `v_g_c`, it should be fine to include here. But that would also mean you can't have (and shouldn't need) [this change](https://github.com/flutter/packages/pull/8014/files#diff-ae3c1055bc547ec174edea702c972431d2d3d46fb9bd78a31378cacf0fd051eaR25) in the PR.